C++/13-Bit Manipulation/1 counting the number of set bits.cpp

```diff
@@ -0,0 +1,25 @@
+	#include<bits/stdc++.h>
```

Comment:
Please change the file names to something like the below:
`1-file_name.cpp`
`2-file_name.cpp`
etc